channels/channel.streams/slam/chn_slam.py

```diff
@@ -0,0 +1,211 @@
+        slam_fm.media_type = mediatype.AUDIO
+        slam_fm.isLive = True
+        slam_fm.add_stream(slam_fm.url)
+        slam_fm.complete = True
+        items.append(slam_fm)
+
+        slam_fm = MediaItem("Slam! The Boom Room", "https://20873.live.streamtheworld.com/WEB12_MP3_SC"
+                                        "?ttag=PLAYER%3ANOPREROLL&tdsdk=js-2.9"
+                                        "&pname=TDSdk&pversion=2.9&banners=none")
+        slam_fm.media_type = mediatype.AUDIO
+        slam_fm.isLive = True
+        slam_fm.add_stream(slam_fm.url)
+        slam_fm.complete = True
+        items.append(slam_fm)
+
+        slam_fm = MediaItem("Slam! 00s", "https://22713.live.streamtheworld.com/WEB15_MP3_SC"
+                                        "?ttag=PLAYER%3ANOPREROLL&tdsdk=js-2.9"
+                                        "&pname=TDSdk&pversion=2.9&banners=none")
+        slam_fm.media_type = mediatype.AUDIO
+        slam_fm.isLive = True
+        slam_fm.add_stream(slam_fm.url)
+        slam_fm.complete = True
+        items.append(slam_fm)
+
+        slam_fm = MediaItem("Slam! Juize", "https://22543.live.streamtheworld.com/WEB09_MP3_SC"
+                                        "?ttag=PLAYER%3ANOPREROLL&tdsdk=js-2.9"
+                                        "&pname=TDSdk&pversion=2.9&banners=none")
+        slam_fm.media_type = mediatype.AUDIO
+        slam_fm.isLive = True
+        slam_fm.add_stream(slam_fm.url)
+        slam_fm.complete = True
+        items.append(slam_fm)
+
+
+        slam_fm = MediaItem("Slam! MixMarathon", "https://stream.slam.nl/web13_mp3"
+                                        "?ttag=PLAYER%3ANOPREROLL&tdsdk=js-2.9"
+                                        "&pname=TDSdk&pversion=2.9&banners=none")
+        slam_fm.media_type = mediatype.AUDIO
+        slam_fm.isLive = True
+        slam_fm.add_stream(slam_fm.url)
+        slam_fm.complete = True
+        items.append(slam_fm)
+
+        slam_fm = MediaItem("Slam! Hardstyle", "https://22323.live.streamtheworld.com/WEB11_MP3_SC"
+                                        "?ttag=PLAYER%3ANOPREROLL&tdsdk=js-2.9"
+                                        "&pname=TDSdk&pversion=2.9&banners=none")
+        slam_fm.media_type = mediatype.AUDIO
+        slam_fm.isLive = True
+        slam_fm.add_stream(slam_fm.url)
+        slam_fm.complete = True
+        items.append(slam_fm)
+
+        slam_fm = MediaItem("Slam! Housuh in de Pauzuh", "http://22553.live.streamtheworld.com/SLAM_MP3_SC"
+                                        "?ttag=PLAYER%3ANOPREROLL&tdsdk=js-2.9"
+                                        "&pname=TDSdk&pversion=2.9&banners=none")
+        slam_fm.media_type = mediatype.AUDIO
+        slam_fm.isLive = True
+        slam_fm.add_stream(slam_fm.url)
+        slam_fm.complete = True
+        items.append(slam_fm)
+        return data, items
+
+   
+
+        Logger.trace(result_set)
+        
+
+    
+
+    def update_live_stream_m3u8(self, item):
```

Comment:
Just a single `update_live_stream` is need that handles the XML content from https://playerservices.streamtheworld.com/api/livestream?station=web14&transports=http%2Chls%2Chlsts&version=1.9 urls.